frontend/src/v5/store/federations/federations.types.ts

```diff
@@ -75,7 +76,7 @@ export type FederationSettings = Omit<FederationBackendSettings, 'surveyPoints'>
 
 export type MinimumFederation = Pick<IFederation, '_id' | 'name' | 'role' | 'isFavourite'>;
 
-export type FederationRevision = Pick<FederationBackendSettings, '_id' | 'timestamp' >;
+export type FederationRevision = Pick<FederationBackendSettings, '_id' | 'timestamp' | 'author' > & {tag: string};
```

Comment:
Is 'name' in FederationBackendSettings just supposed to be Tag? Or is this some legacy v4 code where it has a different name?